posthog/settings/ingestion.py

```diff
@@ -17,3 +17,9 @@
 
 # Keep in sync with plugin-server
 EVENTS_DEAD_LETTER_QUEUE_STATSD_METRIC = "events_added_to_dead_letter_queue"
+
+PARTITION_KEY_AUTOMATIC_OVERRIDE_ENABLED = get_from_env(
+    "PARTITION_KEY_AUTOMATIC_OVERRIDE_ENABLED", type_cast=bool, default=False
+)
+PARTITION_KEY_BUCKET_CAPACITY = get_from_env("PARTITION_KEY_BUCKET_CAPACITY", type_cast=int, default=1000)
+PARTITION_KEY_BUCKET_REPLENTISH_RATE = get_from_env("PARTITION_KEY_BUCKET_REPLENTISH_RATE", type_cast=int, default=1)
```

Comment:
We might need a rate lower than 1 on prod-us, when we have a hundred pods, let's `type_cast=float` here